packages/schemas/tables/users.sql

```diff
@@ -1,6 +1,6 @@
 /* init_order = 1 */
 
-create type users_password_encryption_method as enum ('Argon2i');
+create type users_password_encryption_method as enum ('Argon2i', 'sha1', 'sha256', 'md5');
```

Comment:
```suggestion
create type users_password_encryption_method as enum ('Argon2i', 'SHA1', 'SHA256', 'MD5');
```

we should also support bcrypt?